lizard_auth_server/views.py

```diff
@@ -211,6 +231,20 @@ def form_valid(self, form):
                 kwargs={'invitation_pk': inv.pk}))
 
 
+class ConfirmDeletionUserconsentView(DeleteView):
+    model = UserConsent
+    template_name = 'lizard_auth_server/confirm_deletion_userconsent.html'
+    context_object_name = 'userconsent'
+    success_url = reverse_lazy('profile')
```

Comment:
Prima dat je een "lazy" hebt gebruikt. 

Maandag maar even kort over kletsen, er zijn meer gevallen waarin dat inderdaad nodig is. Handig om wat voorbeelden te zien.